pkg/provider/kubernetes/ingress/fixtures/Ingress-with-port-invalid-for-one-service_ingress.yml

```diff
@@ -0,0 +1,15 @@
+kind: Ingress
+apiVersion: networking.k8s.io/v1beta1
```

Comment:
Can we use the `networking.k8s.io/v1` version please?

Reply:
could we please skip this change if it is just a nitpick? I took a quick look on the requested change and it looks like the whole testing is not ready for `networking/v1`...

Reply:
Okay, in that case, lets skip it :)